sdk/python/foundation-models/benchmarking/evaluating_oai_models.ipynb

```diff
@@ -0,0 +1,199 @@
+{
+ "cells": [
```

Comment:
Does it have to be two different notebooks or just one?

Reply:
The setup instructions and code are different for Claude vs. OAI, so I'd prefer to keep two so that each notebook can stay relatively simple.